numpy/lib/function_base.py

```diff
@@ -1974,7 +1974,7 @@ class vectorize:
     References
     ----------
     .. [1] NumPy Reference, section `Generalized Universal Function API
```

Comment:
```suggestion
    .. [1] :doc:`/reference/c-api/generalized-ufuncs`
```
This should do the trick for an internal link - the url below can be deleted if the doc role is used here.